build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -322,6 +323,107 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        
+        [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CompileUwpHosts(BuildTargetContext c)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I dont think this check is required since BuildPlatform attribute ensures this is invoked for Windows only.